x-pack/plugins/cloud_security_posture/server/create_transforms/create_transforms.ts

```diff
@@ -102,3 +105,17 @@ export const startTransformIfNotStarted = async (
     logger.error(`Failed to check if transform ${transformId} is started: ${statsError.message}`);
   }
 };
+
+const deleteTransformSafe = async (esClient: ElasticsearchClient, logger: Logger, name: string) => {
+  try {
+    await esClient.transform.deleteTransform({ transform_id: name, force: true });
+    logger.info(`Deleted transform successfully [Name: ${name}]`);
+  } catch (e) {
+    if (e instanceof errors.ResponseError && e.statusCode === 404) {
+      logger.trace(`Transform no longer exists [Name: ${name}]`);
```

Comment:
since we're calling `deleteTransformSafe` even when we don't have a transform yet, these logs may get busy and somewhat not needed?

Reply:
that is why I chose to use `trace` logging level - which is the most verbose logging mode and is mostly used for development/ debugging purposes